test/Conversion/TcpToArith/basic.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: tcp-opt %s -convert-tcp-to-arith -split-input-file | FileCheck %s
+// RUN: tcp-opt %s -convert-tcp-to-arith | FileCheck %s
```

Comment:
minor: Why remove `-split-input-file`? Are you expecting this file to not have more than 1 test?

Reply:
Yeah just a cleanup for consistency. Let's add this back when we have more tests per file.